src/backend/distributed/master/master_stage_protocol.c

```diff
@@ -633,6 +633,13 @@ WorkerCreateShard(Oid relationId, int shardIndex, uint64 shardId, List *ddlComma
 		{
 			referencedShardId = shardId;
 		}
+		else if (PartitionMethod(referencedRelationId) == DISTRIBUTE_BY_NONE)
```

Comment:
minor: Can you add a small explanation to the above comment about this case as well?